### ISSUE TITLE
Shrink docker image

### ISSUE THREAD

**Issue by lukemalinowski** [mitre/heimdall-mongo#63]
Work to shrink docker image to 300MB uncompressed. 
Don't know if it's doable, but its a target to shoot for.

**Comment by rbclark** [comment]
I don't think it is actually possible to get much lower than its current state (470MB) with the current gems that are installed. The layer containing the code and gems is 290MB and I don't believe it can be slimmed much more. https://github.com/mitre/heimdall_activerecord/pull/6 is my best attempt at a final slimming.

**Comment by rbclark** [comment]
![Screen Shot 2019-05-28 at 2 50 52 PM](https://user-images.githubusercontent.com/2308869/58504381-7285fe80-8158-11e9-9fe6-ea7f8d729688.png)
